content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -13,7 +13,11 @@ You can set variables specifically for each workspace or you can create variable
 
 ~> **Note:** Variable sets are in beta.
 
-You must have [`read variables` permission](/docs/cloud/users-teams-organizations/permissions.html#general-workspace-permissions) to view the variables for a particular workspace and to view the variable sets in your organization. Once you have the proper [read and write variables permissions](/docs/cloud/users-teams-organizations/permissions.html#general-workspace-permissions), you can create and edit both workspace-specific variables and variable sets through:
+You must have [`read variables` permission](/docs/cloud/users-teams-organizations/permissions.html#general-workspace-permissions) to view the variables for a particular workspace and to view the variable sets in your organization. 
+
+To create and apply a variable set, you need to have either **organization owner** or **workspace manager** permission. 
```

Comment:
Do they need to be added to the "owners" or "workspace manager" team? I want to be as explicit as possible about how they get the right permissions!

Reply:
yeah great feedback!

The team themselves matters less, as long as the user is in a team with "Manage Workspace" permission checked, or in an "owner team", defined as a team with admin permissions (everything is checked under "Organizational Access")

Reply:
<img width="1352" alt="Screen Shot 2021-11-18 at 3 28 23 PM" src="https://user-images.githubusercontent.com/3489963/142512894-73cac064-69d4-4ca7-a059-167fea580b71.png">
to visualize that -- this would be the page to set those permissions

Reply:
(just committed a new version!)